ui.frontend/src/view/FormFieldBase.js

```diff
@@ -152,7 +152,7 @@ export default class FormFieldBase extends FormField {
      * @param enabled
      * @private
      */
-    _updateEnabled(enabled) {
+    _updateEnabled(enabled, readOnly) {
```

Comment:
when invoked from subscribe, readOnly will not be passed.
pls test this when rule editor is used to change enabled property on select type elements.

Reply:
I tested by changing enabled property using rule editor, and when readonly is marked true in the dialog, the children items are getting disabled as intended.

Reply:
I think @nit23uec meant that when the property enabled is changed from a rule (like button click, or anything else) this function will be invoked without readOnly parameter.

Reply:
In subscribe function we are passing `state` as parameter, so the function is invoked along with the `readOnly` parameter, as I had update the function definition in FormFieldBase.js, please refer the screenshot below, and let me know if still there is a gap in my understanding.
![Screenshot (502)](https://user-images.githubusercontent.com/110378842/203474107-a831520d-d814-470e-ba70-25a6e3b3fcd9.png)